src/java.base/share/classes/java/nio/file/Path.java

```diff
@@ -248,6 +249,62 @@ public static Path of(URI uri) {
      */
     Path getFileName();
 
+    /**
+     * Returns the file name extension of this path as a {@code String}. The
+     * extension is the portion of the {@linkplain #getFileName file name}
+     * string after the last period character ('.', U+002E FULL STOP). Leading
+     * period characters in the string are ignored. If there is no  extension,
+     * then {@code null} is returned. This will occur if the path has zero
+     * elements ({@link #getFileName()} returns {@code null}), the file name
```

Comment:
The path has zero elements is correct but it might be clearer to say "The path doesn't have a file contain".

Reply:
> The path has zero elements is correct but it might be clearer to say "The path doesn't have a file contain".

Do you intend "the path does not have a file name"?

Reply:
> Do you intend "the path does not have a file name"?

Yes, I think that would be better than "has zero elements".